static/templates/search_link_404.html

```diff
@@ -0,0 +1,6 @@
+<!-- Printing the search link for the article that wasnt found / a full search for the article. -->
+{{#details}}
+<p>
+	Make a full text search for <a href="{{root}}/search?content={{bookName}}&pattern={{details}}">{{details}}</a>
+</p>
+{{/details}}
```

Comment:
Variable names should reflect their meaning. Please rename `details` to `pattern` or `searchPattern`.